### DISCUSSION TITLE
Bandwidth calibration

### DISCUSSION THREAD

**Discussion by Mauri740** [dl9rdz/rdz_ttgo_sonde#389] [Q&A]
I don't know if the bandwidth calibration I did is correct. I have an RXBX parameter set to default 6300, I have an AGCBW set to default 12500 and I have a frequency offset "-2.5K" which I corrected to set "-2500" now offset is right at about "+0.06". To modify AGCBW I do the sum 6300+2500= 8800. I therefore set the value 10400 instead of 12500. Is this correct? Thank you

**Comment by dl9rdz** [comment]
There is no easy answer and no right or wrong here.

The frequency offset is for calibrating the frequency of the TTGO. Your approach is good for that.

The AGCBW on the other hand is just a trade of between two factors:
- If it is wider than the RS41 signal, you may loos a bit of sensitivity. For that smaller is better (as long as it is larger than the signal bandwidth)
- It is also relevant in case if frequency drifts, of two kinds: Your TTGO may drift in frequency (in particular if you use it mobile outside with higher temperature changes), and the remote sonde might have some frequency error as well. Any of the two causes can result in a mismatch between your frequency and the remote frequency, and if the AGCBW is too small you might have difficulties with reception. For that a larger value may actually be better.

I usually use the default configuration with 12500 Hz but if you know that a sonde is spot on and you want to optimize for receiving a low signal then 10400 might be slightly better. But the difference is not big.
